src/test/java/org/jenkinsci/plugins/registry/notification/CoordinatorTest.java

```diff
@@ -131,6 +138,10 @@ public void testOneTriggered() throws Exception {
         assertNotNull(target_url);
         assertThat(target_url, containsString("/dockerhub-webhook/details/"));
         String sha = target_url.substring(target_url.lastIndexOf('/')+1);
+
+        // the TriggerStore store is provided when the job runs, at the moment all we have guaranteed is a trigger, so we need to wait.
```

Comment:
was not observed to be flaky but the same applies, thus makes sense to prevent it whilst I was here.